raiden-cli/package.json

```diff
@@ -27,14 +27,7 @@
     "@types/node": "^14.14.6",
```

Comment:
I'd say it's better not to:
I'd like the packages to be as standalone as possible, specially if they're published as is and someone try to install them from a registry like `npmjs`, they should be usable standalone. Although `@types/node`  is only required at development time, its depended types may still be required to be available even when installed from registry and used in a TS/ESM project, therefore they should stay on the workspaces. 
Meanwhile, linting rules already (now) depend on the workspace tree (since most of the rules are now in the root), and therefore, the specific devDependencies required for it may go to the root, but I'd say we should keep it at a minimum and let yarn hoisting do its job if/when needed.

Reply:
Okay, so only dev-dependencies should be shared via root?

Reply:
Yes, only devDependencies and which depends on something explicitly requiring the workspace tree (CI, linting, tools, etc). devDependencies used only on package development or dependant's development should stay in the workspace dependencies, as they are now.

Reply:
> For dev-deps I don't get the registry argument.

There are some devDependencies which are there because they are not required at runtime (imported, indirectly run, etc), but which are required if you are developing a project which depends on this package, and have installed it from the registry: e.g. some types; I want to have any devDependency which match this criteria to stay in the workspace's package.json; actually, I want most devDependencies to stay there, and only move up things which already require the workspace tree explicitly, like things which depend on shared tools/configs/etc.